src/DynamoCoreWpf/Views/Gallery/GalleryView.xaml

```diff
@@ -0,0 +1,135 @@
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </UserControl.Resources>
+
+    <UserControl.InputBindings>
+        <KeyBinding Key="Escape"             
+                    Command="{Binding Path=CloseGalleryCommand}" />
+    </UserControl.InputBindings>
+    
+    <Grid>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="100"></ColumnDefinition>
+            <ColumnDefinition Width="*"></ColumnDefinition>
+            <ColumnDefinition Width="100"></ColumnDefinition>
+        </Grid.ColumnDefinitions>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="50"></RowDefinition>
+            <RowDefinition Height="300"></RowDefinition>
+            <RowDefinition Height="50"></RowDefinition>
+            <RowDefinition Height="50"></RowDefinition>
+            <RowDefinition Height="*"></RowDefinition>
+        </Grid.RowDefinitions>
+
+        <TextBox x:Name="Version"
+                 Text="{Binding Path=DynamoVersion, Mode=OneWay}"
+                 Foreground="Gray" FontSize="14" BorderThickness="0"
+                 HorizontalAlignment="Left"
+                 VerticalAlignment="Bottom"
+                 Margin="5,0,0,5"
+                 Grid.Column="1" Grid.Row="0"
+                 FontFamily="{StaticResource OpenSansRegular}"/>
+
+        <Button x:Name="CloseButton"
+                Content="{x:Static p:Resources.GalleryCloseButton}" FontSize="14" Foreground="Red"
+                FontFamily="{StaticResource OpenSansRegular}"
+                HorizontalAlignment="Right"
+                Margin="0,0,5,5"
+                Grid.Column="1" Grid.Row="0"
+                Background="Transparent"
+                BorderThickness="0"
+                Style="{StaticResource {x:Static ToolBar.ButtonStyleKey}}"
+                VerticalAlignment="Bottom" 
+                Command="{Binding Path=CloseGalleryCommand}">
+        </Button>
+
+        <Button x:Name="LeftButton" Focusable="False"
+                HorizontalAlignment="Center" 
+                VerticalAlignment="Center" 
+                Grid.Column="0" Grid.Row="1"
+                Background="Transparent" 
+                BorderThickness="0"
+                Style="{StaticResource {x:Static ToolBar.ButtonStyleKey}}"
+                Width="100"
+                Height="300"
+                Command="{Binding Path=MovePrevCommand}" >
+            <Image Source="LeftIcon.png" 
+                   VerticalAlignment="Center" 
+                   HorizontalAlignment="Center" />
+        </Button>
+
+        <Image x:Name="currentImage" 
+               Source="{Binding Path=CurrentImagePath, UpdateSourceTrigger=PropertyChanged}"
+               Stretch="UniformToFill"
+               Width="400" Height="300" 
+               HorizontalAlignment="Center" 
+               Grid.Column="1" 
+               Grid.Row="1"/>
```

Comment:
Yeah, @elayabharath said that I can assume that the size of the images is always 600x400, (btw this xaml file is not updated with the latest icon). Thanks for the feedback, @Benglin